### ISSUE TITLE
test_publish.R is skipped silently in its entirety

### ISSUE THREAD

**Issue by hlapp** [ropensci/RNeXML#178]
The `test_publish.R` code is wrapped in `if(0)`, i.e., it's never run. That's bad in multiple ways, one of which is that it's not even reported as  a test that's being skipped even though effectively it is.

Perhaps this really isn't currently fit to run as a test anywhere, but even then it should use `skip()` so that it's at least reported as a test that is being skipped.

**Comment by cboettig** [comment]
yeah, it should use skip.  Though really I think we should deprecate the `publish()` methods. Uploading NeXML file to figshare doesn't seem like a particularly effective way to share NeXML. What do you think?

**Comment by hlapp** [comment]
> Though really I think we should deprecate the `publish()` methods.

I agree, actually. I think it should simply be out of scope for this package. Looking at the code, there's not that much special stuff that's being done. _At most_ I'd argue this should be a vignette for how one _might_ upload NeXML data to Figshare (because obviously there are other ways to massage and deposit metadata). But even that might be out of scope, and if anywhere such a vignette should arguably be part of `rfigshare`.

**Comment by cboettig** [comment]
:+1:
